setup.py

```diff
@@ -18,5 +27,9 @@
     license='MIT',
     packages=['formfyxer'],
     install_requires=['spacy',  'PyPDF2',  'pikepdf',  'textstat',  'requests',  'numpy',  'sklearn', 'joblib',  'nltk', 'boxdetect', 'pdf2image', 'reportlab', 'pdfminer.six', 'opencv-python'],
+    cmdclass={
+      'install': InstallSpacyModelCommand,
```

Comment:
This is great! Only question: did we confirm that we can do something like and still be hosted by PyPI?

Reply:
Resolved as of v0.0.8